src/vs/workbench/browser/parts/notifications/media/notificationsToasts.css

```diff
@@ -5,7 +5,7 @@
 
 .monaco-workbench > .notifications-toasts {
 	position: absolute;
-	z-index: 1000;
+	z-index: 9999;
```

Comment:
all edit s in `vs` need the `// {{SQL CARBON EDIT}}` tag.  Please add to all `vs` edits.